src/sql/workbench/browser/editor/profiler/dashboardInput.ts

```diff
@@ -93,6 +94,28 @@ export class DashboardInput extends EditorInput {
 		return name;
 	}
 
+	public override getTitle(verbosity?: Verbosity): string {
+		let baseName = this.connectionProfile.serverName;
+		if (this.connectionProfile.databaseName && !this.isMasterMssql()) {
+			// Only add DB name if this is a non-default, non-master connection and if there is no user set profile name.
+			baseName = baseName + ':' + this.connectionProfile.databaseName;
+		}
+		let advancedOptions = this._connectionService.getNonDefaultOptions(this.connectionProfile);
+		let fullTitle = baseName + advancedOptions;
+
+		switch (verbosity) {
```

Comment:
Why do we have 3 verbosity levels if we there are only two that are effectively used?

Reply:
I will remove the medium option and use the short one as default.